recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
     - pip
   run:
     - python >=3.5
-    - pymysql >=0.9.0,<=0.9.2
+    - pymysql >=0.9.0,<=0.9.3
```

Comment:
To match https://github.com/aio-libs/aiomysql/blob/0a8af2355562ea961477738d66b3334332447186/setup.py#L7.